dkc/core/tests/test_file.py

```diff
@@ -20,7 +20,9 @@ def test_file_rest_retrieve(api_client, file):
 
 @pytest.mark.django_db
 def test_file_sibling_names_unique(file, file_factory):
-    sibling = file_factory.build(folder=file.folder, name=file.name, creator=None)
+    sibling = file_factory.build(folder=file.folder, name=file.name)
+    # Make sure foreign key references exist in database first
+    sibling.creator.save()
```

Comment:
Can we just add the `user` fixture as a dependency and say `creator=user`? Seems a bit more elegant

Reply:
I had it that way before, but it was conceptually helpful to distinguish factory parameters where are conceptually essential to the test case, from foreign key references which must be saved for purely technical reasons.

I liked this better, but I don't feel strongly.